### PR TITLE
Staging/master sync

### PR DESCRIPTION
**Pull request by llewellyn-sl**

### CODE REVIEW

File: docs/wave.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+description: 'Using Wave'
+---
+
+Since version 22.4.x Tower supports Seqera's container augmentation service known as Wave also for on-premises installations.
```
Comment (llewellyn-sl) [inline docs/wave.md:5]:
```suggestion
From version 22.4.0, Tower supports Seqera's Wave containers service for on-premise installations. 
```

File: docs/wave.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+description: 'Using Wave'
+---
+
+Since version 22.4.x Tower supports Seqera's container augmentation service known as Wave also for on-premises installations. 
+You can read more about its features [here](https://wave.seqera.io) and its integration with Nextflow [here](https://www.nextflow.io/docs/latest/wave.html)
```
Comment (llewellyn-sl) [inline docs/wave.md:6]:
```suggestion
See Wave's features [here](https://wave.seqera.io), and Wave integration with Nextflow [here](https://www.nextflow.io/docs/latest/wave.html).
```

File: docs/wave.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+
+## Pairing Tower with Wave
+
+In order to activate the paring mechanism with Wave there are some prerequisites to be fullfilled:
+- https://wave.seqera.io domain should be reachable by Tower network meaning that the domain should be whitelisted in case Tower installation is protected
```
Comment (llewellyn-sl) [inline docs/wave.md:12]:
```suggestion
- https://wave.seqera.io domain should be accessible to the Tower network (i.e. the domain should be whitelisted in protected Tower installations)
```

File: docs/wave.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- The tower installation should be available for inbound traffic coming from wave service
```
Comment (llewellyn-sl) [inline docs/wave.md:13]:
```suggestion
- The tower installation should be available for inbound traffic from the Wave service
```

File: docs/wave.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+- The environment variable `TOWER_ENABLE_WAVE` should be set to the value `true`
```
Comment (llewellyn-sl) [inline docs/wave.md:14]:
```suggestion
- The `TOWER_ENABLE_WAVE` environment variable should be set to `true`
```

File: docs/wave.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+Once the requisites above are fullfilled the wave functionality will be available for activation directly in Tower UI (feature available only for AWS)
```
Comment (llewellyn-sl) [inline docs/wave.md:16]:
```suggestion
When these conditions are met, the Wave feature is available on the Tower compute environment creation page (available for AWS compute environments only)
```

File: docs/wave.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+Once wave is enabled a secure channel will be established between Tower installation and Wave service to securely exchange encrypted data.
```
Comment (llewellyn-sl) [inline docs/wave.md:18]:
```suggestion
Once Wave is enabled, a secure channel is established between Tower and the Wave service to exchange encrypted data securely.
```

File: docs/wave.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+Wave features could also be used by activating wave in the nextflow pipeline config, please read more [here](https://www.nextflow.io/docs/latest/wave.html)
```
Comment (llewellyn-sl) [inline docs/wave.md:20]:
```suggestion
Wave can also be enabled in the Nextflow pipeline config file. See [here](https://www.nextflow.io/docs/latest/wave.html) for more information.
```

File: docs/wave.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+From version 22.4.0, Tower supports Seqera's Wave containers service for on-premise installations. 
+See Wave's features [here](https://wave.seqera.io), and Wave integration with Nextflow [here](https://www.nextflow.io/docs/latest/wave.html).
+
+
+## Pairing Tower with Wave
+
+Pairing Tower with Wave requires the following:
+- `https://wave.seqera.io` should be accessible to the Tower network (i.e. the domain should be whitelisted in protected Tower installations)
+- The Tower installation should allow ingress traffic from the Wave service
+- The `TOWER_ENABLE_WAVE` environment variable must be set to `true`
```
Comment (llewellyn-sl) [inline docs/wave.md:14]:
```suggestion
- The `TOWER_ENABLE_WAVE` environment variable must be set to `true`, or the `enable_wave` parameter set to `true` in your tower.yml configuration
```

File: mkdocs.yml [hunk 10]
```diff
@@ -148,8 +148,9 @@ nav:
     - 'Supported software':
       - 'Illumina DRAGEN': 'supported_software/dragen/overview.md'
       - 'Tower Agent': 'agent.md'
+      - 'Seqera Wave': 'wave.md'
```
Comment (llewellyn-sl) [inline mkdocs.yml:151]:
```suggestion
     - 'Seqera Wave': 'wave.md'
```

**Review by llewellyn-sl — COMMENTED** [review]
Edited Wave content. Syncing staging to master for a big set of updates.

**Review by gwright99 — APPROVED** [review]
Approving so @llewellyn-sl can get updates out to the public website in support of the 22.4.0 release.

**Pull request merged**